src/containers/App/themeConstants.js

```diff
@@ -0,0 +1,44 @@
+export const AP_THEME = {
+  DARK_BLUE: '#141F3C',
+  DARK_BLUE_MEDIUM: '#182444',
+  DARK_BLUE_LIGHT: '#1E2D53',
+  CYAN: '#19C6C7',
+  GRAY: '#B9BCC4',
+  LIGHT_GRAY: '#F0F0F0',
+  WHITE: '#FFFFFF',
+};
+
+export const CONTENT_BACKGROUND = AP_THEME.DARK_BLUE;
+
+export const FILE_BAR = {
+  OPTION_BUTTONS: AP_THEME.WHITE,
+  OPTION_BUTTONS_HOVER: AP_THEME.CYAN,
+  BACKGROUND: AP_THEME.DARK_BLUE_LIGHT,
+};
+
+export const LEFT_NAV = {
+  HEADING: AP_THEME.GRAY,
+};
+
+export const CONTRACT_EDITOR = {
+  BUTTON_BACKGROUND_INACTIVE: AP_THEME.DARK_BLUE_LIGHT,
+  BUTTON_BACKGROUND_ACTIVE: '#364C77',
+  BUTTON_BACKGROUND_HOVER: '#364C77',
+  BUTTON_SYMBOL_INACTIVE: '#949CA2', // #747B8F
```

Comment:
not sure what the commented hex value is for

Reply:
Ah yes, residual from setting this all up.